### PR TITLE
Adding a Readme file

### PR DESCRIPTION
**Pull request by anjapsson**
I did a change

### CODE REVIEW

File: READEME.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+#Heading level 2
```
Comment (adamhagglund) [inline READEME.md]: Pleace add a space between the # and text

Comment (adamhagglund) [reply READEME.md]: And maybe change heading to Welcome

**Review by adamhagglund — CHANGES_REQUESTED** [review]

File: READEME.md (hunk 1 above)
Comment (anjapsson) [reply READEME.md:1]: No thanks

**Review by adamhagglund — APPROVED** [review]
Looks good 👍 Approved

**Pull request merged**